prql-compiler/src/ast/item.rs

```diff
@@ -38,6 +39,28 @@ pub enum Item {
     Windowed(Windowed),
 }
 
+impl Item {
+    pub fn parse_version(self) -> std::result::Result<VersionReq, Self> {
+        match self {
+            Self::Literal(Literal::String(ref s)) => VersionReq::parse(s).map_err(|_| self),
+            Self::Literal(Literal::Integer(major)) => {
```

Comment:
Does this currently accept a bare int like `version:0`? (sorry if I'm being slow at reading the code)

I'm not sure whether we should have the version as a string or bare, but I would vote to only accept one.

Reply:
yes, we basically accept either a string representing a semver version requirement or an integer representing the major version number.